packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,210 @@ class CupertinoIcons {
+  ///
+  /// See also:
+  ///
+  ///  * [switch_camera], which is similar, but not filled in.
+  static const IconData switch_camera_solid = IconData(0xf49f, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections_solid], which is similar, but filled in.
+  static const IconData collections = IconData(0xf3c9, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A collection of envelopes (or folders), which store collections of files. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [collections], which is similar, but not filled in.
+  static const IconData collections_solid = IconData(0xf3ca, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope (or folder), which stores multiple files. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [folder_solid], which is similar, but filled in.
+  ///  * [folder_open], which is the pre-iOS 7 version of this icon.
+  static const IconData folder = IconData(0xf434, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope (or folder), which stores multiple files. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [folder], which is similar, but not filled in.
+  ///  * [folder_open], which is the pre-iOS 7 version of this icon and not filled in.
+  static const IconData folder_solid = IconData(0xf435, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A single envelope that indicates being opened. A folder like this typically stores multiple files.
+  ///
+  /// See also:
+  ///
+  ///  * [folder], which is the equivalent of this icon for iOS versions later than or equal to iOS 7.
+  static const IconData folder_open = IconData(0xf38a, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A trash bin for removing items. This icon is not filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [delete_solid], which is similar, but filled in.
+  static const IconData delete = IconData(0xf4c4, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A trash bin for removing items. This icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [delete], which is similar, but not filled in.
+  static const IconData delete_solid = IconData(0xf4c5, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A trash bin with minimal detail for removing items.
+  ///
+  /// See also:
+  ///
+  ///  * [delete], which is the iOS 7 equivalent of this icon with richer detail.
+  static const IconData delete_simple = IconData(0xf37f, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A pen for composing.
+  ///
+  /// See also:
+  ///
+  ///  * [create_simple], which has less detail and looks like a pencil.
+  static const IconData create = IconData(0xf2bf, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
Describe what it is rather than what it's for since that can change or can be different things for different people looking at it. I would say F417 is the traditional iOS create icon. Just call this write or pen or something.